fabtests/configure.ac

```diff
@@ -80,6 +80,10 @@ AS_IF([test -z "$CFLAGS"],
 # <3), but it is necessary in AM 1.12.x.
 m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 
+dnl Call the provider's CONFIGURE and CONDITIONALS macros
+m4_ifnblank(m4_esyscmd(ls [prov/efa/configure.m4] 2> /dev/null),
```

Comment:
Since configure.m4 is added at the same time, any reason the existence check is needed?

Reply:
nope, removed